test/EndToEnd/Tests/Framework/Script.cs

```diff
@@ -0,0 +1,53 @@
+namespace EndToEndTests.Framework
+{
+    using System.Diagnostics;
+
+    public class Script
```

Comment:
Please make this class internal.

This is only for tests, so may not be required, but we can think of sanitizing the user inputs.

Reply:
This DLL is not published, so the Script class is only for XUnit tests to use. 

Yes, that's why skipped any sanitization checks for now, we can add it as the test suite evolves. Let me know if you still want me to add it.